### PR TITLE
Various fixes. Mainly changing relative links to absolute links

### PR DESCRIPTION
**Pull request by Graeme-Miller**
Changed some relative links to absolute links.

I have run a link checker on the docs with and without these changes. Without these changes there is one more broken link.

### CODE REVIEW

File: guide/ops/production-installation.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ Check that the server meets the [requirements](requirements.html).
 Then configure the server as follows:
 
 * install Java JRE or JDK (version 7 or later)
-* install an [SSH key]({{ site.path.guide }}/ops/locations/index.html#ssh-keys), if not available
+* install an [SSH key]({{ site.path.guide }}/ops/locations#ssh-keys), if not available
```
Comment (drigodwin) [inline guide/ops/production-installation.md]: Why remove this index.html but not the one on the line below?

File: guide/yaml/winrm/index.md [hunk 2]
```diff
@@ -402,7 +402,7 @@ on the Microsoft TechNet site.
 Troubleshooting
 ---------------
 
-Much of the [operations troubleshooting guide](../../ops/troubleshooting/) is applicable for Windows blueprints.  
+Much of the [operations troubleshooting guide](../{{ site.path.guide }}/ops/troubleshooting/) is applicable for Windows blueprints.
```
Comment (drigodwin) [inline guide/yaml/winrm/index.md]: This shouldn't have a `..` at the start

**Comment by Graeme-Miller** [comment]
I am going to make the changes suggested by Duncan to the branch- do not merge yet as I will be testing these changes (I need them pushed so that amp-docs can build correctly)

**Comment by Graeme-Miller** [comment]
Ok, I have tested with the fixes and it looks good. I have removed the production-installation changes, as they are not required in the downstream project any more and I have fixed the accidental relative link.

@aledsage can you merge this?

**Comment by aledsage** [comment]
LGTM; merging now.

**Pull request merged**